macleann/enchiridion-server
# Testing suite implementation

# Testing Suite Implementation

Another massive PR, and this one has been a long time coming. All models and serializers have their respective unit tests. All views have a respective integration test. Updated the README and Jenkinsfile to reflect the new inclusion of the testing suite.

### Bug fixes:
N/A

## Type of change


# How to Test?

Make sure you've followed all installation and setup instructions in the [README](https://github.com/macleann/enchiridion-server/blob/main/README.md).

```
git fetch origin nm-testing
git checkout nm-testing
python manage.py test
```

- [ ] All tests should pass provided setup instructions have been followed.

# Checklist:
